script/Ciel/Day 13/13_01_02_CIEL13_7.txt

```diff
@@ -311,7 +311,7 @@ Even though I stand at death's door,
 -- 「……何がおかしいのよ。先に頭の方がいかれちゃった？　諦めないのは脳が腐って、もうゾンビになってるからなワケ？」
 "What the fuck are you laughing about?
  Did your mind end up breaking first?
- You wouldn't give up because your brain had already rotted away and turned you into a zombie; is that it?"
+ You wouldn't give up because your brain had already rotted away and turned you into a zombie, is that it?"
```

Comment:
Feel like the tense here is a bit weird
As for the semi-colon, I'd rather use another separator.

>You're not giving up because your brain has already rotted away and turned you in a zombie? Is that it?